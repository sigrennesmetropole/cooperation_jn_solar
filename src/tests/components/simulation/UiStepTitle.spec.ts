import UiStepTitle from '@/components/simulation/UiStepTitle.vue'
import { describe, it, expect, beforeEach } from 'vitest'
import { mount, VueWrapper } from '@vue/test-utils'

describe('UiStepTitle.vue', () => {
  let wrapper: VueWrapper

  beforeEach(async () => {
    wrapper = mount(UiStepTitle, {
      propsData: {
        imageSource: 'installation.svg',
        title: 'Votre installation',
      },
    })
  })

  it('renders the component correctly', () => {
    expect(wrapper.exists()).toBe(true)
    expect(wrapper.html()).toMatchSnapshot()
  })
})
