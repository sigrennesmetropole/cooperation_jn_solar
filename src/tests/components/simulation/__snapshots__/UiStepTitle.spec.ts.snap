// Vitest Snapshot v1

exports[`UiStepTitle.vue > renders the component correctly 1`] = `
"<div class=\\"flex flex-col items-center p-0 gap-1\\"><img src=\\"installation.svg\\" class=\\"h-[88px] w-[88px]\\" alt=\\"\\">
  <h1 class=\\"font-dm-sans font-bold text-[30px] leading-9 text-center\\">Votre installation</h1>
</div>"
`;
